terraform/load-balancer/outputs.tf

```diff
@@ -0,0 +1,6 @@
+output "id" {
+  value = (
+    length(azurerm_lb.adl_lb) > 0 ?
+    azurerm_lb.adl_lb[0].id : ""
+  )
+}
```

Comment:
For the sake of consistency with all other modules, let's add name and resource_group name as additional outputs.

`output "name" {
  value = (
    length(azurerm_lb.adl_lb) > 0 ?
    azurerm_lb.adl_lb[0].name: ""
  )
}

output "resource_group_name" {
  value = (
    length(azurerm_lb.adl_lb) > 0 ?
    azurerm_lb.adl_lb[0].resource_group_name: ""
  )
}`